lib/utils.js

```diff
@@ -106,10 +128,19 @@ const unsetConfig = (options, config) => {
     }, {});
     const newValues = (config ? Object.keys(config) : []);
     const diff = oldValues.filter(value => newValues.indexOf(value) < 0).filter(value => value);
-    const message = diff.length === 0 ? 'Nothing has' : (diff.length > 1 ? diff.join(', ') + ' have' : diff[0] + ' has');
+    
+    var message = 'been deleted!';
+    if (diff.length === 0) { message = `Nothing has ${message}`; }
+    else if (diff.length === 1) { message = `${diff[0]} has ${message}`; }
+    else if (diff.length === 2) { message = `${diff[0]} and ${diff[1]} have ${message}`; }
+    else {
+        const last = diff.pop();
+        message = `${diff.join(', ')}, and ${last} have ${message}`;
+    }
+    
```

Comment:
This is the (virtually) the exact same logic as lines 71-77. As a hater of code duplication I'd encourage you to pull this out into a common function and (in this line) just append  `' been deleted'` at the end.

Reply:
Updated! Created new function for this!